AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -71,7 +71,7 @@ - (void)setUp {
   [MSAppCenter resetSharedInstance];
 
   // Mock NSUserDefaults.
-  self.settingsMock = [MSMockUserDefaults new];
+  self.settingsMock = OCMPartialMock([MSMockUserDefaults new]);
```

Comment:
Why do you mock this?? If we want to mock it then we should mock `NSUserDefaults` instead of this mock class. I would suggest you to implement `migrateKeys:forService:` method in `MSMockUserDefaults` class that just set a boolean property to true and assert this value in your test.
I can see that you want to verify migration completed, this will do what you want.

Reply:
Yeah, good point